considerations.md

```diff
@@ -21,13 +21,17 @@ Implementations processing content SHOULD NOT generate an error if they encounte
 - To allow efficient storage, implementations serializing content for blobs SHOULD use a canonical serialization.
 - This increases the chance that different implementations can push the same semantic content to the store without creating redundant blobs.
 
-### JSON
+## JSON
 
-[JSON][JSON] content SHOULD be serialized as [canonical JSON][canonical-json].
 Of the [OCI Image Format Specification media types](media-types.md), all the types ending in `+json` contain JSON content.
-Implementations:
+All [JSON][JSON] content MUST follow the I-JSON limitations in [RFC 7493][rfc7493]:
 
-- [Go][Go]: [github.com/docker/go][docker-go], which claims to implement [canonical JSON][canonical-json] except for Unicode normalization.
+- The encoding MUST be [UTF-8][rfc3629]
+- Numbers SHOULD be limited to IEEE 754 double precision
+- Objects MUST NOT contain duplicate names
+
+The order of entries in JSON objects is not significant.
+Implementations MAY implement Canonical JSON documented in [RFC 8785][rfc8785].
```

Comment:
I believe this is pulling in an unrelated commit.

Reply:
fixed